### PR TITLE
add mod-noShadow to pageHeader

### PR DESCRIPTION
**Pull request by almonais**
Add pageHeader mod-noShadow #1473

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1526

File: stories/qa/page-header/page-header.stories.html [hunk 1]
```diff
@@ -142,3 +142,40 @@ <h1 class="u-marginReset u-marginRightStandard">H1. Page title</h1>
 		</nav>
 	</header>
 </section>
+
+<section class="contentSection">
+	<h2>Header mod-noShadow </h2>
+	<header class="pageHeader mod-noShadow ">
```
Comment (AlexiaAlarza) [inline stories/qa/page-header/page-header.stories.html:148]:
```suggestion
	<header class="pageHeader mod-noShadow">
```

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1526

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1526

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1526

**Review by vvalentin-lucca — APPROVED** [review]

**Pull request merged**